README.md

```diff
@@ -378,11 +378,21 @@ mvn dependency:copy-dependencies
 
 * Configure Kafka Connectors with following properties
 
+When configuring the worker or connectors in Kafka Connect, refer to the string constant properties of the [AWSSchemaRegistryConstants](https://github.com/awslabs/aws-glue-schema-registry/blob/master/common/src/main/java/com/amazonaws/services/schemaregistry/utils/AWSSchemaRegistryConstants.java#L20) class.
+
+Examples:
+
+AWSSchemaRegistryConstants.AWS_REGION = "region"
+AWSSchemaRegistryConstants.REGISTRY_NAME = "registry.name"
+AWSSchemaRegistryConstants.COMPATIBILITY_SETTING = "compatibility";
+
 ```java
     key.converter=com.amazonaws.services.schemaregistry.kafkaconnect.AWSKafkaAvroConverter
     value.converter=com.amazonaws.services.schemaregistry.kafkaconnect.AWSKafkaAvroConverter
     key.converter.region=ca-central-1
+    key.converter.registry.name=my-registry
```

Comment:
Can you please add a comment above this line to indicate that this is optional? We want users to get started with minimal possible configuration.

Reply:
The registry name examples were copied from some of the other configuration examples above in the file. I was not aware of the intent to keep this as an example of the minimal possible configuration. However, if that is the goal here seems to be inconsistency. This led me to assume specifying a specific GSR resource/endpoint wasn't implemented for Kafka Connect. I believe this was because a) I couldn't get it to work with the property names I used (that I mostly copied from the other examples). b) you usually need a URL to the schema registry in Kafka Connect, so it isn't obvious what to do here. In that sense, I added an additional note about this scenario. Maybe that makes more sense.

Please take another look. Also, let me know if I should squash.

Reply:
It would be great if you can squash the commits.

Reply:
Done